src/sentry/api/event_search.py

```diff
@@ -222,10 +222,18 @@ def visit_search_term(self, node, children):
         return search_term[0]
 
     def visit_raw_search(self, node, children):
+        value = node.text
+
+        if value.startswith('"') and value.endswith('"') and value != '"':
+            value = value.strip('"')
```

Comment:
Note that this will strip multiple successive quotes, so `""hello"` translates to `hello`. Maybe fine, just wanted to mention.

Reply:
Added a "balanced" stripper. LMK if while loop is considered dangerous.